app/models/admin/user.rb

```diff
@@ -4,6 +4,8 @@ class User < ApplicationRecord
     validates :username, presence: true, uniqueness: true,
                          length: { minimum: 4, maximum: 30 }, username: true
     validates :name, :department, :role_ids, presence: true, on: :create
+    validates :name, format: { with: /\A[a-zA-Z][a-zA-Z ]+\z/, message: 'must consist of letters and spaces only' }
```

Comment:
Don't forget names can have apostrophes in them - such as Fred O'Donnel

Reply:
and just have a good think about any other characters that you might need to allow for peoples names - maybe a '-' for double barrelled names - unsure